qiskit_ibm_runtime/fake_provider/local_service.py

```diff
@@ -103,12 +103,18 @@ def backends(
         err = QiskitBackendNotFoundError("No backend matches the criteria.")
 
         if name:
-            for b in backends:
-                if b.name == name:
-                    backends = [b]
-                    break
+            if name == "aer_simulator":
```

Comment:
Is there no constant for the AerSimulator name?

Reply:
We don't currently have one, but I can add a constant

Reply:
I'm not sure if this is the correct place to define that constant. It should be in `qiskit-aer` repo.